File: app/src/main/java/com/magnitudestudios/GameFace/pojo/IceCandidatePOJO.java
package com.magnitudestudios.GameFace.pojo;

public class IceCandidatePOJO {
    public String sdpMid;
    public int sdpMLineIndex;
    public String sdp;
    public String serverUrl;

    public IceCandidatePOJO() {}

    public IceCandidatePOJO(String sdpMid, int sdpMLineIndex, String sdp, String serverUrl) {
        this.sdpMid = sdpMid;
        this.sdpMLineIndex = sdpMLineIndex;
        this.sdp = sdp;
        this.serverUrl = serverUrl;
    }
}

File: app/src/main/java/com/magnitudestudios/GameFace/pojo/IceCandidatePOJO.kt
package com.magnitudestudios.GameFace.pojo

data class IceCandidatePOJO (
    @JvmField
    val sdpMid: String = "",
    @JvmField
    val sdpMLineIndex: Int = 0,
    @JvmField
    val sdp: String = "",
    @JvmField
    val serverUrl: String = ""
)
{
    constructor() : this("", 0, "", "")
}